dataflow/gemma/README.md

```diff
@@ -0,0 +1,150 @@
+# RunInference on Dataflow Streaming with Gemma
+
+Gemma is a family of lightweight, state-of-the art open models built from research and technology used to create the Gemini models.
+You can use Gemma models in your Apache Beam inference pipelines with the `RunInference` transform.
+
+This example demonstrates how to utilize a Gemma model in a streaming Dataflow pipeline using Pub/Sub sources and sinks.
+
+For more information about using RunInference, see [Get started with AI/ML pipelines](https://beam.apache.org/documentation/ml/overview/) in the Apache Beam documentation.
+
+## Before you begin
+
+Follow the steps in this section to create the necessary environment to run this workflow.
+
+### Enable Google Cloud services
+
+This workflow uses multiple Google Cloud Platform products, including Dataflow, Pub/Sub, Google Cloud Storage, and Artifact Registry. Before you start the workflow, create a Google Cloud project that has the following services enabled:
+
+* Dataflow
+* Pub/Sub
+* Google Cloud Storage
+* Artifact Registry
+
+Using these services incurs billing charges.
+
+### Create a custom container
+
+You need to use Docker to build a custom container. This repository contains a Dockerfile that you can use to build your custom container. Follow the [Google Cloud documentation](https://cloud.google.com/dataflow/docs/guides/build-container-image#build_and_push_the_image) that explains how to build and push a container to Artifact Registry by using Docker.
+
+### Create Pub/Sub topics for input and output
+
+Follow the [Google Cloud documentation for creating Pub/Sub topics](https://cloud.google.com/pubsub/docs/create-topic#pubsub_create_topic-Console). This example uses two: one input topic and one output topic.
+
+### Download and save the model
+
+Save a version of the Gemma 2B model. Downloaded the model from [Kaggle](https://www.kaggle.com/models/keras/gemma/frameworks/keras/variations/gemma_2b_en), and then rename the downloaded archive to `gemma_2B`.
+
+### Import dependencies
+
+Install Apache Beam and the dependencies required to run the pipeline in your local environment. **Note that the Python major and minor version contained in the custom container must match the environment used for job submission.**
```

Comment:
I'd say: example require python3.11 and add a command that checks the local python version is indeed 3.11. Another option is to also provide a flex template to submit the job, then template will control the python version.

Reply:
I added a call-out for the versions, we could drop a command to check here if desired